### PR TITLE
fix: add fallpack for undefined env var

### PR DESCRIPTION
**Pull request by danielleadams**
#### Description of changes
Fixes a parsing error that happens when running e2e tests locally due to a missing environment variable that is supplied by CI.

#### Checklist

- [x] PR description included
- [x] `yarn test` passes

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by sobolk — DISMISSED** [review]

**Review by jhockett — DISMISSED** [review]

**Comment by danielleadams** [comment]
Failing test is unrelated to changes.

**Comment by danielleadams** [comment]
@sobolk @jhockett this needs another review because there was a merge conflict

File: packages/amplify-e2e-core/src/utils/index.ts [hunk 1]
```diff
@@ -30,7 +30,8 @@ export * from './git-operations';
 /**
  * Whether the current environment is CircleCI or not
  */
-export const isCI = (): boolean => process.env.CI && process.env.CIRCLECI && JSON.parse(process.env.CI) && JSON.parse(process.env.CIRCLECI);
+
+export const isCI = (): boolean => JSON.parse(process.env.CI || 'false') && JSON.parse(process.env.CIRCLECI || 'false');
```
Comment (jhockett) [inline packages/amplify-e2e-core/src/utils/index.ts:34]: Is this change still needed? Looks like `isCI` was updated to handle the cases where these variables are undefined.

Comment (danielleadams) [reply packages/amplify-e2e-core/src/utils/index.ts:34]: The previous statement returns undefined when `process.env.CI` and `process.env.CIRCLECI` are undefined. The above statement will always return a boolean, which is the defined return type. Typescript compiler doesn't seem to mind. I'm partial to returning the correct type for better definition of the function, but I don't feel too strongly since the compiler passes.

**Review by sobolk — APPROVED** [review]

**Review by jhockett — APPROVED** [review]
